020-Connect/010-Postgres/001-direct-access.mdx

```diff
@@ -138,66 +138,88 @@ Xata offers Postgres in a variety of [pricing formats](/pricing). Shared cluster
 
 The table below provides full granularity over specific statements.
 
```

Comment:
This diff is not very helpful 😂

Reply:
yeah, this is not ideal... I think since I added a long statement name, the linter then extended all of the first column for every row